src/modules/sensors/voted_sensors_update.h

```diff
@@ -140,7 +140,7 @@ class VotedSensorsUpdate
 	/**
 	 * Calculates the magnitude in Gauss of the largest difference between the primary and any other magnetometers
 	 */
-	void calc_mag_inconsistency(sensor_preflight_s &preflt);
+	void calcMagInconsistency(sensor_preflight_s &preflt);
```

Comment:
Can you keep this consistent with the others (calc_accel_inconsistency, calc_gyro_inconsistency)? Updating them all or not.

Reply:
Yeah, I thought about switching all of them but the diff gets huge... let me still do it.

Reply:
I refactored all `voted_sensors_update` functions to camelCase and consider this solved.